book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,173 @@
+An interesting model to consider emulating here is [moodle](https://moodle.com/) an open source electronic learning environment popular with universities.
+It offers commercial support and hosting and has a network of developers across the education sector who contribute to the core project as well as creating custom integrations and extensions to meet their particular needs.
+This structure allows institutions to pool resources to fund the development of features that they all need and keeps this infrastructural code in common ownership, avoiding the worst lock-in issues arising from the use of proprietary solutions.
+
+(rr-rdm-open-elns)=
+## Select Open Infrastructure ELN Options
+
+They are each quite different but have many of the same core features.
+For example rich text editing in a web browser.
+The ability to upload files.
+Sharing and permissions based on roles/groups.
+
+*  [eLabFTW](https://www.elabftw.net/) 
+    *   Laboratory resource scheduling feature for booking things like hoods and microscopes, automatic mol file previews for molecules and proteins & support for free-hand drawing.
+    *   The eLabFTW [site](https://www.elabftw.net/) and [documentation](https://doc.elabftw.net/) there is also a [demo](https://demo.elabftw.net/login.php) deployment that you can try out
+    *   Self-hosting is *relatively* simple [according to the documentation](https://doc.elabftw.net/install.html).
+    There is also a [paid support tier](https://www.deltablot.com/elabftw/) which would be recommend for any larger deployment to support the ongoing development of the project.
+    *   [Paid cloud hosting](https://www.deltablot.com/elabftw/) is available from the developer in a geographical region suited to your needs, a more expensive tier with hosting in France compliant with additional security and privacy certifications is available.
+* [openBIS](https://openbis.ch/)
+    *   Good features for integrated metadata management for example linking to ontologies / controlled vocabularies.
+    This is based in a flexible object system for making similar entries.
+    *   openBIS has an API and can integrate with jupyterhub for electronic lab notebooks.
+    *   Very feature rich LIMS system with optional integration of stores management with protocols and experiments including keeping track of bar-coded stocks.
+    *   You can get a feel for it in the [demo](https://openbis-eln-lims.ethz.ch/openbis/webapp/eln-lims/) deployment.
+    *   openBIS is a bit more complex to administer [based on its documentation](https://openbis.ch/index.php/docs/admin-documentation/).
+    It's a slightly older and more complex project than the others on this list, meaning it is very featureful and well tested against the needs of the groups at ETH Zurich.
+    *   Developed at ETH Zurich openBIS can be hosted for you under the openRDM service operated by ETH Zurich scientific IT services.
```

Comment:
```suggestion
    *   As openBIS is developed at ETH Zurich it can be hosted for you under the openRDM service operated by ETH Zurich scientific IT services.
```